src/main/java/edu/neu/EDE/io/WorkbookReader.java

```diff
@@ -0,0 +1,92 @@
+package edu.neu.EDE.io;
+
+import org.apache.poi.ss.usermodel.Cell;
+import org.apache.poi.ss.usermodel.Row;
+import org.apache.poi.xssf.usermodel.XSSFSheet;
+import org.apache.poi.xssf.usermodel.XSSFWorkbook;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+
+/**
+ * Created by Ian on 2/26/15.
+ */
+public class WorkbookReader {
+
+    final int SHEET_MEDIA_NAME_ROW = 0;
+    final int SHEET_MEDIA_NAME_CELL = 5;
+    final int SLIDE_METRIC_ROW_START = 2;
+    final int STATISTIC_NAME_CELL = 0;
+    final int STATISTIC_VALUE_CELL = 5;
+    String subject;
+    String media;
+
+    public void readFile(File f) throws IOException {
```

Comment:
Once data issue is resolved, this will be readFile(File f, 4darray slideMetricData, 4darray lookZoneData)